src/request.js

```diff
@@ -1,9 +1,26 @@
 import _ from 'underscore';
 import util from 'util';
+import crypto from 'crypto';
 import { checkParamOrThrow } from 'apify-client/build/utils';
 import { normalizeUrl } from 'apify-shared/utilities';
+import log from 'apify-shared/log';
 
-export const computeUniqueKey = (url, keepUrlFragment) => normalizeUrl(url, keepUrlFragment);
+export function computeUniqueKey({ url, method, payload, keepUrlFragment }) {
+    const normalizedMethod = method.toUpperCase();
+    const normalizedUrl = normalizeUrl(url, keepUrlFragment);
+    if (normalizedMethod === 'GET') return normalizedUrl;
```

Comment:
This is a breaking change actually, and CHANGELOG.md says nothing. Actually, I wouldn't do this by default - it will make practically every crawler that uses POST requests to run infinitely, which is not a great default behavior. I'd make it an option

Reply:
We talked about it with @mtrunkat and it's only a breaking change for 0.1% of users so we decided to go with it (the CHANGELOG is an omission), because without it, enqueueing of POST requests is pretty much unusable for users. Not only they need to understand the the intricacies of `uniqueKeys`, but they also need to compute a unique key themselves wherever they enqueue new requests, which could be at a number of places.